packages/moode-player/build.sh

```diff
@@ -24,10 +24,10 @@ NPM_CI=0
 # build web app with gulp, to speed up test build without change to frontend (or manual build) diable this
 BUILD_APP=1
 
-GULP_BIN=$MOODE_DIR/node_modules/.bin/gulp
+GULP_BIN="${MOODE_DIR}/node_modules/.bin/gulp"
 
 # Used as reference for generating station patch files. Should be the first releas of a major
-MAJOR_BASE_STATIONS=../dist/binary/moode-stations-full_8.0.0.zip
+MAJOR_BASE_STATIONS="moode-stations-full_8.0.0.zip"
```

Comment:
This one should be reverted the stations file should be in the binary output (is the output of the first 8.0.0 build)

Reply:
nvm Ok I see you added later, but maybe at least add a var to the directory next to it.

Reply:
added a var holding the dist/binary/ directory string and referenced it in the appropiate locations